src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/StaffRefDataController.java

```diff
@@ -19,4 +39,61 @@ public class StaffRefDataController {
     @Value("${loggingComponentName}")
     private String loggingComponentName;
 
+    @Autowired
+    StaffRefDataService staffRefDataService;
+
+    @ApiOperation(
+            value = "This API gets the user types from staff reference data",
+            authorizations = {
+                    @Authorization(value = "ServiceAuthorization"),
+                    @Authorization(value = "Authorization")
+            })
+    @ApiResponses({
+            @ApiResponse(
+                    code = 200,
+                    message = "Successfully fetched the user types",
+                    response = UserType.class
+            ),
+            @ApiResponse(
+                    code = 401,
+                    message = UNAUTHORIZED_ERROR
+            ),
+            @ApiResponse(
+                    code = 403,
+                    message = FORBIDDEN_ERROR
+            ),
+            @ApiResponse(
+                    code = 404,
+                    message = "The User Types could not be found"
+            ),
+            @ApiResponse(
+                    code = 500,
+                    message = INTERNAL_SERVER_ERROR
+            )
+    })
+    @GetMapping(
+            path = "/user-type",
+            produces = APPLICATION_JSON_VALUE
+    )
+    @Secured("cwd-admin")
+    public ResponseEntity<Object> fetchUserTypes() {
+        log.info("{} : Fetching the user types", loggingComponentName);
+        StaffRefDataUserTypesResponse.StaffRefDataUserTypesResponseBuilder staffReferenceDataUserTypesResponseBuilder
+                = StaffRefDataUserTypesResponse.builder();
+        List<UserType> userTypes = staffRefDataService.fetchUserTypes();
+
+        if (ObjectUtils.isNotEmpty(userTypes)) {
+            List<StaffRefDataUserType> refDataUserTypes = userTypes.stream()
+                    .map(StaffRefDataUserType::new)
+                    .toList();
+            staffReferenceDataUserTypesResponseBuilder.userTypes(refDataUserTypes);
+            log.debug("refDataUserTypes = {}", refDataUserTypes);
+            return ResponseEntity
+                    .status(200)
+                    .body(staffReferenceDataUserTypesResponseBuilder.build());
+        } else {
```

Comment:
do we need to return 404 ? I thought endpoint should return 200 for empty user types as well. check with Sujeeth on this.

Reply:
yes, it should return 200 with empty response